nuxeo-java-client-test/src/test/java/org/nuxeo/client/ITRepository.java

```diff
@@ -955,13 +955,13 @@ public void itCanRetrieveCommentPermissions() {
         comment = commentAdapter.create(comment);
 
         // assert its permissions
-        assertEquals(
-                Arrays.asList("Write", "WriteVersion", "ReadProperties", "ReadCanCollect", "ReadSecurity", "Remove",
-                        "ReadVersion", "Read", "WriteLifeCycle", "Everything", "Moderate", "Version", "ReadChildren",
-                        "AddChildren", "Comment", "ReadLifeCycle", "RemoveChildren", "DataVisualization",
-                        "ReviewParticipant", "Unlock", "CanAskForPublishing", "RestrictedRead", "ReadWrite",
-                        "ReadRemove", "Browse", "WriteProperties", "WriteSecurity", "ManageWorkflows"),
-                comment.getPermissions());
+        assertTrue(comment.getPermissions()
```

Comment:
Could you add a first argument `comment.getPermissions().toString()` so that if there's a failure we see what the actual permissions are in the exception?